CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.5.0] - 2023-05-24
+
+### Added
+* Support for NET 6
+* Support for WebView2 in the WinUI package
+### Changed
+* The WinUI package temporarily uses the Uno version to 4.9.0-dev.1113 instead of 4.0.7
```

Comment:
From what I gather you want to say that the WinUI package is not using a stable version of Uno right?

Reply:
@GabrielChouinardLetourneau There should not be a changed entry for WinUI since it's being added in this PR

Reply:
I will do another PR to address the doc